docs/en/differenze_oidc_fed.rst

```diff
@@ -11,19 +11,13 @@ This section lists the differences between the official standard and the SPID /
 Client Registration
 +++++++++++++++++++
 
-SPID supports only **automatic_client_registration**. The **implicit** flow is not supported.
-
-
-Listing endpoint
-++++++++++++++++
-
-In the SPID / CIE Federations, for this endpoint, the claim **entity_type** is added to all the other `OIDC-FED`_ standard claims, in order to obtain a filter on the type of subordinate Entities. This need apecifically allows filtering Entities of type **federation_entity**, **openid_relying_party**, **openid_provider** and **oauth_resource**.
+SPID and CIE support only **automatic_client_registration**. The **explicit client registration** flow is not supported.
 
 
 Trust Mark
 ++++++++++
 
-In the SPID and CIE id Federations, using Trust Marks is not mandatory. Rather, in the SPID Federation it is mandatory to expose them. For more details about the reasons why the Trust Marks are required, please see the section :ref:`Security Considerations <Considerazioni_di_Sicurezza>`.
+In OIDC Federation international specifications the adoption of the Trust Marks are not mandatory. Rather, in the SPID and CIE id Federation it is mandatory to expose them. For more details about the reasons why the Trust Marks are required, please see the section :ref:`Security Considerations <Considerazioni_di_Sicurezza>`.
```

Comment:
the adoption of the Trust Marks are not mandatory --> the adoption of the Trust Marks is not mandatory

Reply:
done 
https://github.com/italia/spid-cie-oidc-docs/pull/121/commits/04b0df11cbc03c9175fb505cee5a450229949a3d